client/components/LearnerViewDirective.js

```diff
@@ -35,8 +35,10 @@ tie.directive('learnerView', [function() {
                   </div>
                   <div ng-class="{'tie-step-line': $index < (questionIds.length - 1)}"></div>
                 </div>
+                <div ng-class="{'tie-step-line': $index < (questionIds.length - 1)}"></div>
               </div>
             </div>
+          </div>
```

Comment:
I see. I added a </div> by mistake, and then shifted the block below to the left. I'll shift them back.